### PR TITLE
Use the script ci.sh in snc repo instead of hardcoding all steps here

### PR DESCRIPTION
**Pull request by guillaumerose**

### CODE REVIEW

File: ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh [hunk 1]
```diff
@@ -34,29 +34,7 @@ function run-tests() {
...
-
-  # Check the cluster operator output, status for available should be true for all operators
-  while oc get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}' | grep -v True; do
-     sleep 2
-  done
-
-  # Run createdisk script
-  export SNC_VALIDATE_CERT=false
-  ./createdisk.sh crc-tmp-install-data
+  ./ci.sh
```
Comment (cfergeau) [inline ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh:37]: Don't we need to keep 
```
  if [[ $? -ne 0 ]]; then
    exit 1
  fi
```
?

Comment (guillaumerose) [reply ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh:37]: does `set -e` cover us here ? I guess it should but we probably miss `set -e` in ci.sh.

Comment (cfergeau) [reply ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh:37]: Ah right should be good enough. snc.sh uses something more extensive (`set -exuo pipefail`), but `-e` seems like what we need here.

**Review by cfergeau — APPROVED** [review]

**Pull request merged**